src/veh_interact.cpp

```diff
@@ -2441,8 +2445,7 @@ void complete_vehicle ()
 
         broken = veh->parts[vehicle_part].hp <= 0;
         if (!broken) {
-            used_item = veh->parts[vehicle_part].properties_to_item();
-            g->m.add_item_or_charges(g->u.posx(), g->u.posy(), used_item);
+            g->m.add_item_or_charges( g->u.posx(), g->u.posy(), veh->parts[vehicle_part].properties_to_item() );
```

Comment:
If you're changing this line already, you could also make it use the tripoint argument.